src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
 // write code here
+const listOfEmploeesLi = document.querySelectorAll('li');
+const listOfEmploees = [...listOfEmploeesLi];
+
+listOfEmploees.sort((a, b) =>
+  (+b.dataset.salary.substring(1).split(',').join(''))
+  - (+a.dataset.salary.substring(1).split(',').join('')));
```

Comment:
виглядає так собі)
на майбутнє створюй хелпер функцію, яка опише що тут відбувається
```suggestion
  getSalary(a) - getSalary(b);
```